base/all-all/sbardef.lmp

```diff
@@ -930,7 +930,7 @@
           {
             "widget":
             {
-              "x": 0,
+              "x": 2,
```

Comment:
Would it be possible to stringify entries like e.g. `"alignment": 16` somehow? I think this would raise the acceptance of the new lump syntax. 

Something like `"alignment": "left-align"` or similar?

Reply:
Even with these changes, the JSON format will be difficult to edit by hand. And it is not designed for that.

The editor is working, I'm just busy and took a break for a week.